packages/flutter/lib/src/widgets/form.dart

```diff
@@ -349,6 +349,15 @@ class FormFieldState<T> extends State<FormField<T>> {
   /// True if this field has any validation errors.
   bool get hasError => _errorText != null;
 
+  /// True if the current value is valid.
```

Comment:
nit: the first sentence should be its own paragraph. Please add a blank line below this one.

Reply:
Done.